include/boost/core/latch.hpp

```diff
@@ -0,0 +1,104 @@
+//  Copyright 2023 Christian Mazakas
+//  Distributed under the Boost Software License, Version 1.0.
+//  https://www.boost.org/LICENSE_1_0.txt
+
+#ifndef BOOST_CORE_LATCH_HPP_INCLUDED
+#define BOOST_CORE_LATCH_HPP_INCLUDED
+
+#include <boost/config.hpp>
+
+#if defined(BOOST_NO_CXX11_HDR_CONDITION_VARIABLE)
+BOOST_PRAGMA_MESSAGE("boost::core::latch requires <condition_variable>")
```

Comment:
These messages are unnecessary. The inclusion of `<condition_variable>` below is enough to imply the this header requires it.